plugin/semisync/semisync_master.h

```diff
@@ -579,6 +582,10 @@ class ReplSemiSyncMaster : public ReplSemiSyncBase {
    */
   mysql_mutex_t LOCK_binlog_;
 
+  mysql_mutex_t LOCK_last_acked;
+  mysql_cond_t COND_last_acked;
+  std::pair<std::string, my_off_t> last_acked;
```

Comment:
typedef/using for a complex type
```suggestion
using binlog_position_type = std::pair<std::string, my_off_t>;
binlog_position_type last_acked
```